pages/common/find.md

```diff
@@ -33,4 +33,4 @@
 
 - Find empty files (0 byte) or directories and delete them verbosely:
```

Comment:
```suggestion
- Find empty files or directories and delete them verbosely:
```
I feel like the parenthese creates an unnecessary gap before the word "or"

Reply:
Agree - "(0 byte)" is clunky and ugly

Reply:
I like this better "List and delete empty files and directories"